inst/extdata/energy/A21.globaltech_coef.csv

```diff
@@ -0,0 +1,16 @@
+# File: A21.globaltech_coef.csv,,,,,,
```

Comment:
Here and in next file, please remove ugly Excel commas (",,,,,,") at end of header lines